docs/examples/cas9_kymotracking/cas9_kymotracking.rst

```diff
@@ -158,7 +139,7 @@ the longest line we found, and have a look at its position over time::
     longest_line = kymowidget.lines[longest_index]
 
     plt.figure(figsize=(5, 3))
-    plt.plot(np.array(longest_line.time_idx) * dt, np.array(longest_line.coordinate_idx) * pixel_size / 1000)
+    plt.plot(np.array(longest_line.seconds), np.array(longest_line.position))
```

Comment:
I think that you can also drop the `np.array()` casts now.

Reply:
Yeah good catch.